webrtc-stats.html

```diff
@@ -481,9 +481,12 @@ <h3>
               <code><a>RTCOutboundRtpStreamStats</a></code>.
             </p>
             <p>
-              When simulcast is in use, there will be one RtpStreamStats per
+              When simulcast is in use, there will be one
```

Comment:
Rephrase as when multiple SSRCs are used and say "an example of this is simulcast".
This would include RTX and stuff like that